content/events/ai-workshop.mdx

```diff
@@ -0,0 +1,85 @@
+  and increase their everyday productivity with AI. The workshop includes
+  practical applications, starting from text and image generation, then
+  development tools, to building AI assistants.
+_body:
+  - eventDurationInDays: 2
+    price: 440
+    gstText: inc GST
+    eventList:
+      - city: Brisbane
+        date: 2024-03-21T00:00:00.000Z
+        bookingURL: 'https://www.eventbrite.com.au/'
+      - city: Melbourne
+        date: 2024-03-27T00:00:00.000Z
+        bookingURL: 'https://www.eventbrite.com.au/'
+      - city: Sydney
+        date: 2024-04-01T00:00:00.000Z
+        bookingURL: 'https://www.eventbrite.com.au/'
+    _template: EventBooking
+  - header: Agenda
+    textColor: red
+    agendaItemList:
+      - placeholder: Day 1
+        body: |
+          <VerticalListItem
+            content={<>
+              Day 1: Generative AI and LLMs
+            </>}
+            icon="/images/icons/university_hat.png"
+            afterBody={<>
+              * Machine Learning - How models are built and trained
+              * What is generative AI?
+              * Learn text, image, audio, and video generation
+              * ChatGPT, DALL-E, Midjourney, Claude, Llama
+              * Prompt engineering
+              * Tokens, prompt tuning, chain prompting, system prompts
+              * Prompt practice
+              * Understanding custom GPTs
+              * Build a custom GPT
+            </>}
+          />
+      - placeholder: Day 2
+        body: |
+          <VerticalListItem
+            content={<>
+              Day 2: Applications of Generative AI
+            </>}
+            afterBody={<>
+              * GitHub Copilot
+              * Using the OpenAI API
+              * Embeddings
+              * Semantic Kernel
+              * Plugins, planners, semantic memory
+              * Demo app, or similar
+              * Build a custom Assistant with the OpenAI API
+            </>}
+            icon="/images/icons/university_hat.png"
+          />
+    _template: Agenda
+  - header: Presenters
+    presenterList:
+      - presenter: content/presenters/calum-simpson.mdx
+      - presenter: content/presenters/jernej-kavka.mdx
+    _template: PresenterBlock
+  - content: "## Introduction\n\nSSW is thoroughly exploring the evolving field of AI for software development and everyone at SSW is encouraged to use AI in their work. Since Generative AI has become widespread, developers taking advantage of it are building better software faster and SSW’s productivity has risen. This workshop aims to share the knowledge of these great modern tools to help others.\n\n\_\n\n## Who Should Attend\n\nDevelopers of all skill levels with any level of AI experience.\n\n\_\n\n## What’s Included?\n\n\\- Directed learning with hands-on exercises\n\n\\- Mentoring from SSW experts\n\n\\- Source code and project templates for your reference\n\n\\- Catering (Morning tea, lunch & afternoon tea)\n"
```

Comment:
Can you fix the bullet points here, they aren't displaying correctly

Reply:
done